docs/content/v0-upgrade.md

```diff
@@ -0,0 +1,464 @@
+
+### Scenario 6
+
+All consumers can be run without flags, as the bundle will contain
+attributes to inform v1.0 OPAs to accept v0.x modules.
+
+#### How to Run
+
+- Policies are authored to be v0.x compatible.
+- v0.x consumers are run as-is. Bundles will contain v0.x policies
+- v1.0 consumers are run as-is. Bundles will contain `rego_version` attribute, so v0.x modules are accepted.
+
+#### Pre-requisites
+
+If users cannot set their OPA v1 producers to use `--v0-compatible` to be
+compatible with their v0.x consumers, then this upgrade path is blocked.
+
+#### Next
+
+Running exclusively v1.0 producers and consumers, ([Scenario 9](#scenario-9)), is
+the next and final step.
+
+### Scenario 7
+
+All consumers are v1.0, but producers are v0.x. This scenario might occur when
+OPAs used for evaluation are upgraded before the policy bundling system.
+
+#### Pre-requisites
+
+If v1.0 consumers cannot be run with `--v0-compatible`, when loading v0 consumer
+generated bundle, the bundles cannot include `rego_version` attribute. This means
+the upgrade path is blocked until either the consumers can create bundles with a
+Rego version or the `--v0-compatible` flag is available for producers.
+
+#### How to Run
+
+- Policies are authored to be v0.x compatible.
+- v1.0 consumers are run with `--v0-compatible`
+
+#### Next
+
+Upgrade producers to v1.0 ([Scenario 8](#scenario-8)) until all producers are v1.0
+([Scenario 9](#scenario-9)).
+
+### Scenario 8
+
+All consumers are v1.0, but producers are a mix of v0.x and v1.0.
+
+#### How to Run
+
+- Policies are authored to be v0.x compatible.
+- v1.0 consumers are run with `--v0-compatible`
+- v1.0 producers are run with `--v0-compatible`
+- `rego.v0` imports cannot be used; rejected by v0.x bundle producers.
+
+#### Pre-requisites
+
+If using v0 bundles, it must be possible to use `--v0-compatible` on the bundle
+producers in order for them to work in the v1 consumers.
+
+Similarly, on the consumer side. If `--v0-compatible` cannot be set on the
+consumers, the bundles from v1 producers will not be accepted. v0.x bundle
+producers will produce v1.0-incompatible bundles as they will have no Rego
+version attribute.
+
+#### Next
+
+Upgrade producers to v1.0 ([Scenario 9](#scenario-9)), completing the upgrade.
+
+### Scenario 9
+
+Once you have all consumers and producers running at v1.0 then you have
+completed the upgrade to OPA v1.0. If you are using `--v0-compatible`
+functionality, the next task is to upgrade the Rego loaded into OPAs to Rego v1.
+
+Regardless of whether you are now upgrading your Rego, we encourage users to
+use `opa check`, `opa check --strict` and to lint their Rego projects if you
+have not already done so to identify issues.
+
+## Changes to Rego in OPA v1.0
+
+Once you have upgraded OPA instance to v1.0, or if you are upgrading all at
+once, you will need to upgrade your Rego policies to Rego v1. This section
+outlines the changes in Rego v1. See
```

Comment:
> See ...

Missing link?

Reply:
Nah..I think a result of copy-paste. Updated.